docs/topics/gradle/gradle-configure.md

```diff
@@ -0,0 +1,745 @@
+[//]: # (title: Configure your existing project)
+
+To build a Kotlin project with [Gradle](https://docs.gradle.org/current/userguide/getting_started.html), 
+you'll need to add the [Kotlin Gradle plugin](#apply-the-plugin) to your build script file `build.gradle(.kts)` 
+and [configure dependencies](#configure-dependencies) there. As the prerequisite, you need to make sure that you have a Gradle Wrapper in 
```

Comment:
```suggestion
and [configure any dependencies](#configure-dependencies). As a prerequisite, you need to make sure that you either, have a Gradle Wrapper in 
```

Reply:
Well, these dependencies are not *any*, they are some particular dependencies depending on what a user wants.

Reply:
@sarahhaggarty, you might have missed my answer.

Reply:
OK. What about "your" instead of "any"?

Reply:
Sounds strange) They are project dependencies.
I changed to "configure the project's dependencies".